.github/workflows/pypi-publish.yml

```diff
@@ -0,0 +1,29 @@
+name: Publish package to PyPi
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  push:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.5
```

Comment:
Can we run this on 3.8 instead?

Reply:
Updated to deploy on python 3.8 instead.